lib/tasks/manageiq/ui_tasks.rake

```diff
@@ -39,11 +39,23 @@ if Rake::Task.task_defined?("assets:precompile")
   Rake::Task["assets:precompile"].enhance do
     Rake::Task["webpack:compile"].invoke
   end
+
+  Rake::Task["assets:precompile"].actions.each do |action|
+    if action.source_location[0].include?(File.join("lib", "tasks", "webpacker"))
+      Rake::Task["assets:precompile"].actions.delete action
```

Comment:
> Does it make sense to warn or something in an else block so we know when this is fixed/changed?

Not sure what you mean.  There are you saying the outer if block checking to see if `assets:precompile` exists?

> I feel a bit dirty removing someone else's action... In what circumstance does their action make sense?

When the source of the [`bin/webpacker` and `bin/webpacker-dev-server`](https://github.com/ManageIQ/manageiq-ui-classic/tree/master/bin) are in the root of the Rails project.  They are not in our setup.

> oh boy @NickLaMuro, @bdunne's unseattle will not like the `.delete action`

Oh... right... will change.

Update:  Oh snap!  [Rubocop didn't complain](https://github.com/ManageIQ/manageiq-ui-classic/pull/1860#issuecomment-321002582) :trollface: